pkg/manager/rollbackconfig.go

```diff
@@ -20,31 +20,38 @@ import (
 	networkchange "github.com/onosproject/onos-api/go/onos/config/change/network"
 	networkchangestore "github.com/onosproject/onos-config/pkg/store/change/network"
 	"github.com/onosproject/onos-config/pkg/store/stream"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 )
 
 // RollbackTargetConfig rollbacks the last change for a given configuration on the target, by setting phase to
 // rollback and state to pending.
 func (m *Manager) RollbackTargetConfig(networkChangeID networkchange.ID) error {
 
+	if networkChangeID == "" {
+		return status.Error(codes.InvalidArgument, "error on rollback. networkChangeID is empty")
```

Comment:
I think we should use onos-lib-go error package which has the ability to convert typed errors to gRPC errors and vice versa.

Reply:
@adibrastegarnia hopefully I got the ones you meant

Reply:
@SeanCondon thanks